### PR TITLE
Align the DE/book structure to the EN original

### PR DESCRIPTION
**Pull request by petrisch**
So I guess its time to get this in another shape.
This is not finished yet as I think the chapter files should be made first.
But I wanted to show you that I am on this.

### CODE REVIEW

**Comment by petrisch** [comment]
I wonder what happened to the mathematik.md?

**Comment by amtoine** [comment]
thanks @petrisch :pray: 

i'll mark this as draft for the time being until this PR is ready :wink:

**Comment by hustcer** [comment]
> I wonder what happened to the mathematik.md?

`book\math.md` was removed in b019eabe2251e33f5da791b68e353cf165fc2e7b

**Comment by petrisch** [comment]
So this is probably going to be it for the next 3 weeks since I will not have that much time left.
What do you think is this in a mergable stage already?
I have probably made a lot of mistakes, so if someone ca review it would be cool

**Comment by sholderbach** [comment]
Thanks for working on this! I think by aligning it to the structure of the current English book this should make it easier for folks to jump in with translations or reference back to the original if necessary. With the translations we are still in the best effort phase as the development outpaces the English reference which outpaces the translations.

So I would be up to landing it, when you are comfortable with it.

File: de/book/advanced.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# (Nicht nur für) Fortgeschrittene
+
+Der Titel "Fortgeschritten" mag abschrecken und zum Überspringen des Kapitels animieren, doch tatsächlich sind einige der interessantesten und mächtigsten Merkmale hier zu finden.
+
+Abgesehen von den eingebauten Befehlen, besitzt Nushell einen [Standard ibliothek](standard_library.md).
```
Comment (sholderbach) [inline de/book/advanced.md:5]:
```suggestion
Abgesehen von den eingebauten Befehlen, besitzt Nushell eine [Standard Bibliothek](standard_library.md) aus in Nu geschriebenen Befehlen.
```

File: de/book/advanced.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Nushell arbeitet mit _strukturierten Daten_.
+Nushell ist eine "Daten orientierte" Shell und Programmiersprache.
+Diese Daten zentrierte Sicht zeigt sich auch in [Polars](https://github.com/pola-rs/polars), dem eingebauten vollständigen Datenpaket Verarbeiter.
+Im Kapitel [Dataframes documentation](dataframes.md) wird erklärt, wie grosse Datenmengen effizient in Nushell verarbeitet werden können.
+
+Werte in Nusehll enthalten einige [Metadaten](metadata.md).
+Diese können verwendet werden, um z.B. [eigene Fehlermedlungen](creating_errors.md) erstellen zu können.
+
+Dank Nushells strikten Gültigkeits Regeln, ist es sehr einfach durch Sammlungen [parall zu iterieren](parallelism.md), was mit wenig Aufwand lange laufende Skripte verschnellern kann.
```
Comment (sholderbach) [inline de/book/advanced.md:15]:
```suggestion
Diese können verwendet werden, um z.B. [eigene Fehlermeldungen](creating_errors.md) erstellen zu können.

Dank Nushells strikten Gültigkeitsregeln, ist es sehr einfach durch Sammlungen [parallel zu iterieren](parallelism.md), was mit wenig Aufwand lange laufende Skripte beschleunigen kann.
```

**Comment by petrisch** [comment]
@sholderbach if it doesn't brake anything I'm find to land it.
I can't test the website at the moment, so if someone can do this would be nice.

**Comment by sholderbach** [comment]
Ah right now the paths appear as the labels in the menu:

grafik

**Review by sholderbach — APPROVED** [review]
Looking good locally!

**Pull request merged**